teachertool/src/transforms/runEvaluateAsync.ts

```diff
@@ -19,7 +19,7 @@ export async function runEvaluateAsync(fromUserInteraction: boolean) {
     // EvalRequest promises will resolve to true if evaluation completed successfully (regarless of pass/fail).
     // They will only resolve to false if evaluation was unable to complete.
     const evalRequests = teacherTool.checklist.criteria.map(criteriaInstance =>
-        runSingleEvaluateAsync(criteriaInstance.instanceId, false)
+        runSingleEvaluateAsync(criteriaInstance.instanceId, fromUserInteraction)
```

Comment:
This will cause multiple toasts to appear, one for each individual criteria evaluated, and then a final toast once all evaluates.

Reply:
It doesn't do that on success. But I do see that happening when there's an error in the criteria. Will address...

Reply:
Pushed the toast-display code back up to the callers of runSingleEvaluateAsync to handle this